src/Components/LandingPage.js

```diff
@@ -14,14 +14,17 @@ class LandingPage extends Component {
     }
 
     render() {
+        const textColSize = 5;
```

Comment:
Kind of nit, but I would use caps to show they are constants `TEXT_COL_SIZE` etc. 
Not sure if thats a js thing, but it def helps in java to know what's constant.